src/eventHandlers/continuousIntegrationEnd/index.ts

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable no-await-in-loop */
 
 import { context, getOctokit } from '@actions/github';
-import type { GraphQlQueryResponseData } from '@octokit/graphql';
+import { GraphQlQueryResponseData } from '@octokit/graphql';
```

Comment:
Why the change?

Reply:
I think I used it before, not anymore. Fixed.